engine/src/test/scala/cromwell/engine/workflow/tokens/JobExecutionTokenDispenserActorSpec.scala

```diff
@@ -45,7 +70,12 @@ class JobExecutionTokenDispenserActorSpec extends TestKit(ActorSystem("JETDASpec
   }
 
   it should "dispense indefinitely for an infinite token type" in {
-    val senders = (1 to 20).map(_ => TestProbe())
+    val actorRefUnderTest =
+      getActorRefUnderTest(
+        serviceRegistryActorName = "serviceRegistryActor-dispense-indefinitely",
```

Comment:
Full names was originally a TOL but now that I'm seeing it, I'm convinced this is much nicer - thank you!